ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -243,3 +243,9 @@ def compress(condition, a, *, axis=None, out=None):
         )
     arr = arr[: condition_arr.shape[0]]
     return ivy.moveaxis(arr[condition_arr], 0, axis)
+
+
+@inputs_to_ivy_arrays
```

Comment:
You will need to import this function wrapper in the top imports

Reply:
> You will need to import this function wrapper in the top imports

yes, added. Thanks